tests/integration/targets/virtualdirectory/tasks/main.yml

```diff
@@ -0,0 +1,88 @@
+- name: Check if we can run the tests
+  ansible.windows.win_shell: |
+    $osVersion = [Version](Get-Item -LiteralPath "$env:SystemRoot\System32\kernel32.dll").VersionInfo.ProductVersion
+    $osVersion -ge [Version]"6.2"
+  register: run_test
+  changed_when: false
+
+- name: Run on Server 2012 and higher
+  when: run_test.stdout | trim | bool
```

Comment:
Since now the oldest server is 2016 and it does support Get-WindowsFeature, this check can be removed.
```suggestion
```